packages/nc-gui/components/smartsheet/details/Fields.vue

```diff
@@ -1072,7 +1079,7 @@ const onFieldOptionUpdate = () => {
             </Draggable>
           </div>
           <Transition v-if="!changingField" name="slide-fade">
-            <div class="border-gray-200 border-l-1 rounded-r-xl h-[calc(100vh-(var(--topbar-height)*3.85))]">
+            <div class="border-gray-200 border-l-1 rounded-r-xl nc-scrollbar-md nc-fields-height !overflow-y-auto">
```

Comment:
Please explain why these classes are required.

Reply:
**Removed Unwanted classes in below commit** 

[fix : fixed overflow in fieldContainer](https://github.com/nocodb/nocodb/pull/7114/commits/a31b544ba56eac7876bf1e428bac3e1e4000a298)
[a31b544](https://github.com/nocodb/nocodb/pull/7114/commits/a31b544ba56eac7876bf1e428bac3e1e4000a298)